osu.Server.Spectator/Hubs/CountdownImplementation.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using osu.Game.Online.Multiplayer;
+using osu.Game.Online.Multiplayer.Countdown;
+
+namespace osu.Server.Spectator.Hubs
+{
+    /// <summary>
+    /// The implementation for a countdown timer.
+    /// </summary>
+    public class CountdownImplementation
```

Comment:
I find it a touch weird that this is an `Implementation`, since that term is generally one we have used for the case where there are going to be more than one implementations.

I understand why this can't be a `ServerCountdown : Countdown` in a similar fashion to how `ServerMultiplayerRoom` is structured, but still not sure about the naming. `ServerCountdown` may fit better?